scripts/scil_ft_visualize_collisions.py

```diff
@@ -0,0 +1,126 @@
+#! /usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+Visualize collisions found through the intersection filtering process of
+fibertubes (see ft_filter_collisions.py). The previously obtained
+"colliders" tractogram is to be used as input in the present script. The
+collision points need to be stored as data_per_streamline on each of its
+streamlines.
```

Comment:
Can you please finish every help of the "ft" family of scripts by defining what ft means. 

ft := fibertube? Should we use fibertube in the names of the files for clarity? I don't think anybody would now what ft means. In fact, most people may think it is ft := fiber tracking.

Reply:
Good idea, "ft. is for "fibertube tracking", but I think just "fibertube" in filenames would be better. I'll change them right before merging